### PR TITLE
[NO-TICKET] doc site updates

### PR DESCRIPTION
**Pull request by scheul93**
## Summary

* Updating nav urls to include query param from current URL
* Instead of `pushState`, updated to `replaceState` so that user cannot use the back button to access a URL without the theme param in URL
* Fixing a CSS property so it has a valid value

## How to test

`yarn start:gatsby`

## Still TODO
Sometimes, locally, the site will give a 404 on a page that should exist. I think it has something to do with Gatsby client-side routes. This is not something I have been able to reproduce in our beta deployment

### CODE REVIEW

File: packages/docs/src/styles/components/storybookExample.scss [hunk 1]
```diff
@@ -13,7 +13,7 @@
 }
 
 .c-storybook-example-footer {
-  display: relative;
+  position: relative;
```
Comment (pwolfert) [inline packages/docs/src/styles/components/storybookExample.scss:16]: Whoops, that was me 😭

**Review by pwolfert — APPROVED** [review]
It works a lot better now! Nice fix

**Comment by pwolfert** [comment]
https://cmsgov.github.io/design-system/branch/akamm/doc-site-updates

**Review by forrestbaer — APPROVED** [review]
nice!

**Pull request merged**